extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -296,9 +296,14 @@ public void handle(AsyncResult<Void> event) {
             throw new RuntimeException("Unable to start HTTP server", e);
         }
 
-        // TODO log proper message
-        Timing.setHttpServer(String.format(
-                "Listening on: http://%s:%s", httpServerOptions.getHost(), httpServerOptions.getPort()));
+        String serverListeningMessage = String.format(
+                "Listening on: http://%s:%s", httpServerOptions.getHost(), httpServerOptions.getPort());
+
+        if (sslConfig != null) {
+            serverListeningMessage = serverListeningMessage
+                    + String.format(" and on: https://%s:%s", sslConfig.getHost(), sslConfig.getPort());
```

Comment:
```suggestion
                    + String.format(" and https://%s:%s", sslConfig.getHost(), sslConfig.getPort());
```
would be better because a bit shorter, I think. WDYT?